2.1 Inspector Gadget/exercise1.rb

```diff
@@ -12,6 +12,14 @@ def auto_complete(editor)
     if word == "\n"
       throw :eol
     end
+      if editor.template.user_classes.include? word
+        l = eval(word.to_s).methods
```

Comment:
what is `l`?

Reply:
my bad. I will never use one or two-character variable names.